plugins/redis/lib_redis.ecllib

```diff
@@ -17,57 +17,77 @@
 
 
 EXPORT sync := SERVICE : plugin('redis'), namespace('RedisPlugin')
```

Comment:
I wondered this my self and almost changed it to redis, but *if*, for what ever reason we do want to add the async stuff, it would makes sense to keep this as sync now.
Otherwise, no it's not and it's really related to hiredis rather than redis itself.
I'll change to redis.